app/server/utils/user.py

```diff
@@ -845,10 +845,16 @@ def create_user_without_transfer_account(phone):
                 last_name=temporary_last_name,
                 phone=phone,
                 registration_method=RegistrationMethodEnum.USSD_SIGNUP)
+
+    organisation = g.active_organisation
+
+    if organisation:
+        user.add_user_to_organisation(organisation, False)
```

Comment:
this pattern seems to get used a lot. should probably refactor it into the instantiation of a user obj.